deploy-manage/deploy/self-managed/install-kibana-from-archive-on-linux-macos.md

```diff
@@ -38,11 +43,33 @@ cd kibana-{{version.stack}}/ <2>
 
 1. Compares the SHA of the downloaded `.tar.gz` archive and the published checksum, which should output `kibana-<version>-linux-x86_64.tar.gz: OK`.
 2. This directory is known as `$KIBANA_HOME`.
+:::
+
+:::{tab-item} Specific version
+:sync: specific
+Because {{kib}} is an {{stack}} product, you must install the same version number that's used across the stack. Replace `<SPECIFIC.VERSION.NUMBER>` with the {{stack}} version.
```

Comment:
"same version number that's used across the stack" reads a little weird to me

this is an overkill edit perhaps

```suggestion
Because {{kib}} is an {{stack}} product, you need to install the same version number as the rest of your {{stack}} components. Replace `<SPECIFIC.VERSION.NUMBER>` with the version used across your entire stack (for example, {{version.stack.base}}).
```

Reply:
Are we using `need to` instead of `must` to convey the absence of obligation?
If that's the case, I'd say something like `you might want to install the same version` as it is clearer from a translation-ready perspective. 

On the other hand, if we want to convey that using the same version across the stack is an absolute requirement, I'd stick with `must` (which is also a clearer translation-ready option because phrasal verbs, modal verbs, and conditionals do not translate in the same way in every language).

Do we have any guidance on this, do you know?

Reply:
Will stick with `must` to convey the requirement, we've chatted about it in slack. Thank you!